dag.yml

```diff
@@ -1,10 +1,18 @@
 steps:
   data://garden/owid/latest/covid:
     - etag://raw.githubusercontent.com/owid/covid-19-data/master/public/data/owid-covid-data.csv
+  data://garden/owid/latest/population_density:
+    - data://garden/owid/latest/key_indicators
```

Comment:
Would it make sense to move `population_density` to `key_indicators` (possibly together with `land_area`)?